### PR TITLE
Adds historical redirect of /cli -> /install.sh

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Noticed this 404 in analytics and confirmed there are some references to this
https://github.com/search?q=getenvoy.io%2Fcli&type=code

https://app.netlify.com/sites/getenvoy/analytics

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
also added a robots.txt as some of these hits are likely crawlers who may honor it

File: site/static/robots.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+User-agent: *
```
Comment (mathetake) [inline site/static/robots.txt:1]: nice!

File: ci/test [hunk 2]
```diff
@@ -1,7 +1,8 @@
 #! /bin/bash
 
 # Ensure CLI docs match latest GetEnvoy docs
-./ci/regen
+# TODO: temporarily disabled until next release
```
Comment (mathetake) [inline ci/test:4]: should have done in the previous PR #170 but fine now 😄

**Review by mathetake — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
thanks for the help!

**Pull request merged**